packages/nouns-contracts/test/foundry/DAOUpgrade/DAOUpgradeMainnetFork.t.sol

```diff
@@ -0,0 +1,185 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.19;
+
+import 'forge-std/Test.sol';
+import { Strings } from '@openzeppelin/contracts/utils/Strings.sol';
+import { NounsDAOLogicV4 } from '../../../contracts/governance/NounsDAOLogicV4.sol';
+import { ProposeDAOUpgradeMainnet } from '../../../script/DAOUpgrade/ProposeDAOUpgradeMainnet.s.sol';
+import { NounsToken } from '../../../contracts/NounsToken.sol';
+import { INounsDAOLogicV3 } from '../../../contracts/interfaces/INounsDAOLogicV3.sol';
+import { NounsDAOTypes } from '../../../contracts/governance/NounsDAOInterfaces.sol';
+
+abstract contract DAOUpgradeMainnetForkBaseTest is Test {
```

Comment:
@eladmallel CI doesn't run fork tests at the moment.
I think we should make it run them, because we are now running without optimizer locally.
It will probably require setting up some alchemy/infura RPC into the github secrets. we may need soli's help for that

Reply:
I agree it's good to have, let's ask for help!